src/Context/PlaylistContext.js

```diff
@@ -0,0 +1,95 @@
+import axios from "axios";
+import { createContext, useContext, useState, useEffect } from "react";
+import { toast } from "react-toastify";
+const PlaylistContext = createContext()
+const usePlaylist = () => useContext(PlaylistContext)
+const PlaylistContextProvider = ({children}) =>{
+    const [playlist, setPlaylist] = useState([])
+    const [playlistModal, setPlaylistModal] = useState(false)
+    useEffect(()=>{
+        getPlaylist()
+    },[])
```

Comment:
Suggestion: never leave a dependency array empty if there is some external variable/function/object there in useEffect.